sdk/storage/storage-blob/src/Clients.ts

```diff
@@ -283,6 +291,29 @@ export interface BlobExistsOptions extends CommonOptions {
    * @memberof BlobExistsOptions
    */
   customerProvidedKey?: CpkInfo;
+  /**
+   * Conditions to meet.
+   *
+   * @type {BlobRequestConditions}
+   * @memberof BlobExistsOptions
+   */
+  conditions?: BlobRequestConditions;
+  /**
+   * An opaque DateTime value that, when present, specifies the blob
+   * snapshot to retrieve.
+   * 
+   * @type {string}
+   * @memberof BlobExistsOptions
+   */
+  snapshot?: string;
```

Comment:
Interesting, is the swagger lay adding the missing feature for download snapshot?

Reply:
Aligning with `getProperties`. I don't know why we missed it in the first place.
@XiaoningLiu any idea?

Reply:
@jlian Please remove snapshot from options. As for blobClient, there is a `withSnapshot()` method to get a blobClient pointing to snapshot. It's a duplication to add in options again.

Reply:
I don't think I'm the right person to do this?

Reply:
Done.

> Please remove snapshot from options. As for blobClient, there is a `withSnapshot()` method to get a blobClient pointing to snapshot. It's a duplication to add in options again.